test/services/dumpers/agent-nodejs/expected/mongodb/.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+npm-debug.log
+.env
```

Comment:
No extra carriage return at the end of our files ?

Reply:
👍